maintenance/build-bundle.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -e
 
-cat node_modules/@wikia/ad-engine/dist/ad-engine.global.js <(echo) ./dist/ad-products.global.js <(echo) ./dist/geo.global.js > ./dist/global-bundle.js
```

Comment:
It turns out that geo module is already included in `utils`
https://github.com/Wikia/ad-products/blob/dev/src/utils/index.js#L1